### PR TITLE
[9.0][ADD] purchase_double_validation: Migration scripts

### PR DESCRIPTION
**Pull request by pedrobaeza**
@Tecnativa TT20683

### CODE REVIEW

**Comment by StefanRijnhart** [comment]
Unexpected travis error

```
File "/home/travis/build/OCA/OpenUpgrade/addons/purchase/migrations/9.0.1.2/post-migration.py", line 303, in migrate_purchase_double_validation

    words = env.ref('purchase.trans_confirmed_double_gt').condition.split()

  File "/home/travis/build/OCA/OpenUpgrade/openerp/api.py", line 792, in ref

    return self['ir.model.data'].xmlid_to_object(xml_id, raise_if_not_found=raise_if_not_found)

  File "/home/travis/build/OCA/OpenUpgrade/openerp/api.py", line 248, in wrapper

    return new_api(self, *args, **kwargs)

  File "/home/travis/build/OCA/OpenUpgrade/openerp/api.py", line 490, in new_api

    result = method(self._model, cr, uid, *args, **old_kwargs)

  File "/home/travis/build/OCA/OpenUpgrade/openerp/addons/base/ir/ir_model.py", line 1049, in xmlid_to_object

    raise ValueError('No record found for unique ID %s. It may have been deleted.' % (xmlid))

ValueError: No record found for unique ID purchase.trans_confirmed_double_gt. It may have been deleted.
```

File: openerp/openupgrade/doc/source/modules80-90.rst [hunk 1]
```diff
@@ -474,7 +474,7 @@ missing in the new release are marked with |del|.
 +----------------------------------------+------------------------------------------+
 | |del| purchase_analytic_plans          | Moved to OCA [#purchase_analytic_plans]_ |
 +----------------------------------------+------------------------------------------+
-| |del| purchase_double_validation       |                                          |
+| |del| purchase_double_validation       | Done                                     |
```
Comment (StefanRijnhart) [inline openerp/openupgrade/doc/source/modules80-90.rst]: Do we want to change this to `Done (merged into purchase)` or is that redundant?

Comment (pedrobaeza) [reply openerp/openupgrade/doc/source/modules80-90.rst]: Yeah, I think that's the best thing!

**Comment by pedrobaeza** [comment]
Fixed. Workflow is being deleted before, so I need to take the value on pre before this removal.

**Review by StefanRijnhart — APPROVED** [review]
Thanks for the update!

**Pull request merged**